test/baseline/User.ts/User.test.js

```diff
@@ -151,7 +151,26 @@ describe('User.ts/user endpoint test', () => {
     expect(attributes.status).to.eql('ok')
   })
 
-  it('should get user trackgroups by trackgroup id', async () => {
+  //  FIXME: 20221011 mb. Skip.
+  //  Returns error:
+  //    {"status":404,"message":"User is not associated to TrackGroup!","data":null}
```

Comment:
See my comment about the User / UserGroup error. It's the same thing. 

Since the migrations `users` are no longer associated to `TrackGroups`. Instead, now `UserGroups` are associated to `TrackGroups`. So the query in the endpoint has to be updated to reflect this. 

Specifically [this line](https://github.com/resonatecoop/api/blob/main/src/controllers/user/trackgroups/routes/%7Bid%7D/index.js#L267) probably needs to be updated to say `model: UserGroup` and then the `as: 'user'` needs to be removed.